python/cudf/cudf/_lib/pylibcudf/strings/contains.pyx

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+from libcpp.memory cimport unique_ptr
+from libcpp.utility cimport move
+
+from cudf._lib.pylibcudf.column cimport Column
+from cudf._lib.pylibcudf.libcudf.column.column cimport column
+from cudf._lib.pylibcudf.libcudf.strings cimport contains as cpp_contains
+from cudf._lib.pylibcudf.strings.regex_program cimport RegexProgram
+from cudf._lib.pylibcudf.scalar cimport Scalar
+
+from cython.operator import dereference
+
+from cudf._lib.pylibcudf.libcudf.scalar.scalar cimport string_scalar
+
+cpdef Column contains_re(
+    Column input,
+    RegexProgram prog
+):
```

Comment:
docstring needed

Reply:
We might also want to start the sphinx docs for the strings submodule for pylibcudf.

As far as I can tell, there's no .rst files for strings in the api_docs/pylibcudf folder.

Reply:
Added a few sphinx docs here